audformat/core/utils.py

```diff
@@ -376,6 +360,43 @@ def join_labels(
     return joined_labels
 
 
+def join_schemes(
+        dbs: typing.Sequence[Database],
+        scheme_id: str,
+):
+    r"""Join and update scheme of two databases.
+
+    This joins the given scheme of two databases
+    using :func:`audformat.utils.join_labels`
+    and replaces the scheme in each database
+    with the joined one.
+
+    This might be useful,
+    if you want to combine both databases
+    with :meth:`audformat.Database.update`.
+
+    Args:
+        dbs: sequence of databases
+        scheme_id: scheme ID of a scheme with labels
+            that should be joined
+
+    Example:
+        >>> db1 = Database('db1')
+        >>> db2 = Database('db2')
+        >>> db1.schemes['scheme_id'] = Scheme(labels=['a'])
+        >>> db2.schemes['scheme_id'] = Scheme(labels=['b'])
+        >>> join_schemes([db1, db2], 'scheme_id')
+        >>> db1.schemes
+        scheme_id:
+          dtype: str
+          labels: [a, b]
+
+    """
+    labels = join_labels([db.schemes[scheme_id].labels for db in dbs])
```

Comment:
maybe immediately return if `len(dbs) <= 1`?

Reply:
I don't think it will be any speed issue if the code gets executed, or are you worried that the database object is touched with `replace_labels`?

Reply:
Ok, as long as `join_labels` works with an empty list we can also leave it as it is.

Reply:
I added a test for it, it does work.